SparkPost/2web2ui
# FE-933 - Re-style global pie chart components

[FE-933](https://jira.int.messagesystems.com/browse/FE-933)

### What Changed

- [x] Updated `<PieChart/>` component (including child components):
  + [x] `<Legend/>`
  + [x] `<LegendItem/>`
  + [x] `<ActiveLabel/>`
- [x] Updated `<PercentWheel/>` component
- [x] Updated `<Chart/>` component
- **Note:** I decided against updating the colors in the pie charts as these are defined at the feature level rather than the component level. I did update the `<PercentWheel/>` colors as that was lower hanging fruit - ideally this component would provide these colors in a default manner

### How To Test

- To see the pie chart components in action, go to `/reports/bounce`
  + Verify that hover states persist between Hibana and OG versions of the component
  + Verify that clickable `<LegendItem/>` elements are clickable (with a visual indication that they are) and verify that non-clickable `<LegendItem/>` elements are not clickable
- Go to `/inbox-placement/details/<id>/details` - verify the `<PercentWheel/>` components match the new mocks

### To Do
